### DISCUSSION TITLE
`bun list`

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [oven-sh/bun#8263] [Q&A] [answered]
`> bun list`
`Uh-oh. bun list is a subcommand reserved for future use by Bun.`

When? 😭

**Comment by angelhdzmultimedia** [comment, marked as answer]
### Solved
` bun pm ls` 🤦‍♂️😂🤗🍞🐰🔥

**Reply by ianzone** [reply]
why not `bun ls`?

**Reply by angelhdzmultimedia** [reply]
https://tenor.com/bSbQp.gif
